tests/test_core.py

```diff
@@ -7937,6 +7937,10 @@ def make_run(name, emcc_args, env=None):
   if env is None:
     env = {}
 
+  # Filter out the opt level
+  opt_level = [a for a in emcc_args if a.startswith('-O')]
+  emcc_args = [a for a in emcc_args if not a.startswith('-O')]
```

Comment:
I think there may be tests that look at `-O?` in `self.emcc_args`, and this might break them?